lib/main.dart

```diff
@@ -123,7 +144,11 @@ class _AppState extends State<App> {
         return LiveViewBackground(
           child: Stack(
             children: [
-              child!,
+              Listener(
```

Comment:
werkt dit ook nog wanneer de tap/click ook wordt opgevangen door een `GestureDetector` (bijv. de Print-knop, of de QR-knop)?

Reply:
Ja, heb ik getest. Om de een of andere reden niet wanneer je een `GestureDetector` gebruikt ipv de `Listener`, maar dit doet het goed.

Reply:
ja ik denk dat `GestureDetector` de event op-eet voordat andere `GestureDetector`'s erbij kunnen, maar dat `Listener` meer raw is (ik word direct wijzer uit de docs, dit is hoe ik het zelf interpreteer), maar in dat geval is dit uitstekend!